pkg/types/baremetal/platform.go

```diff
@@ -40,10 +40,31 @@ type Platform struct {
 	// +optional
 	ExternalBridge string `json:"externalBridge,omitempty"`
 
-	// Provisioning bridge is used for provisioning nodes.
+	// Provisioning bridge is used for provisioning nodes, on the host that
+	// will run the bootstrap VM.
 	// +optional
 	ProvisioningBridge string `json:"provisioningBridge,omitempty"`
 
+	// ProvisioningNetworkInterface is the name of the network interface on a baremetal server
+	// that it uses to connect to the provisioning network.
```

Comment:
we could perhaps say it's the network interface on the master nodes, not just any baremetal server

Reply:
But it is, isn't it? We'll wire this into the MAO for configuring Metal3, or do we have additional ability to customize the provisioning interface there?

Eventually I think this should just be migrated to be part of each `Host`, instead of assuming homogeneous hardware